terraform/cloud-platform-components/resources/opa/opa-psp-rbac.yaml

```diff
@@ -0,0 +1,16 @@
+# Being allowed to use the _privileged_ policy means binding it to the default serviceaccount of a namespace.
+# A simple RoleBinding resource needs to be created, as described below :
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: PrivilegedRoleBinding
```

Comment:
Can we please rename this to something a bit more descriptive? For example `opa:psp-privileged` or similar

Reply:
Updated name to opa:psp-privileged